client/src/types.ts

```diff
@@ -19,7 +19,6 @@ export interface AxiomV2ClientConfig<T> {
   provider: string;
   callback: AxiomV2CallbackInput;
   privateKey?: string;
-  version?: string;
```

Comment:
Why was this removed?  Same for the `privateKey` field below

Reply:
they're not used and are also not in the docs; i can make a separate PR to remove if preferred

Reply:
OK, this is fine, but in general it makes the PR pretty hard to review / we should explicitly separate cleanup / refactoring from feature additions.